guides/source/caching_with_rails.md

```diff
@@ -286,90 +349,174 @@ simply be explicit in a comment, like:
+
+```bash
+rails new app_name --skip-solid
 ```
 
-NOTE: Notice that in this example we used the `cache_key_with_version` method, so the resulting cache key will be something like `products/233-20140225082222765838000/competing_price`. `cache_key_with_version` generates a string based on the model's class name, `id`, and `updated_at` attributes. This is a common convention and has the benefit of invalidating the cache whenever the product is updated. In general, when you use low-level caching, you need to generate a cache key.
+WARNING: Both Solid Cache and Solid Queue are bundled behind the `--skip-solid` flag. If you still want to use Solid Queue but not Solid Cache, you can enable Solid Queue by running `rails app:enable-solid-queue`.
 
-#### Avoid Caching Instances of Active Record Objects
+### Configuring the database
 
-Consider this example, which stores a list of Active Record objects representing superusers in the cache:
+To use Solid Cache, you can configure the database connection in your `config/database.yml` file. Here's an example configuration for a SQLite database:
 
-```ruby
-# super_admins is an expensive SQL query, so don't run it too often
-Rails.cache.fetch("super_admin_users", expires_in: 12.hours) do
-  User.super_admins.to_a
-end
+```yaml
+production:
+  primary:
+    <<: *default
+    database: storage/production.sqlite3
+  cache:
+    <<: *default
+    database: storage/production_cache.sqlite3
+    migrations_paths: db/cache_migrate
 ```
 
-You should __avoid__ this pattern. Why? Because the instance could change. In production, attributes
-on it could differ, or the record could be deleted. And in development, it works unreliably with
-cache stores that reload code when you make changes.
+In this configuration, the `cache` database is used to store cached data. You can also specify a different database adapter, like MySQL or PostgreSQL, if you prefer.
 
-Instead, cache the ID or some other primitive data type. For example:
+```yaml
+production:
+  primary: &primary_production
+    <<: *default
+    database: app_production
+    username: app
+    password: <%= ENV["APP_DATABASE_PASSWORD"] %>
+  cache:
+    <<: *primary_production
+    database: app_production_cache
+    migrations_paths: db/cache_migrate
+```
 
-```ruby
-# super_admins is an expensive SQL query, so don't run it too often
-ids = Rails.cache.fetch("super_admin_user_ids", expires_in: 12.hours) do
-  User.super_admins.pluck(:id)
-end
-User.where(id: ids).to_a
+If `database` or [`databases`](#sharding-the-cache) is not specified in the cache configuration, Solid Cache will use the ActiveRecord::Base connection pool. This means that cache reads and writes will be part of any wrapping database transaction.
+
+In production, the cache store is configured to use the solid cache store:
+
+```yaml
+  # config/environments/production.rb
+  config.cache_store = :solid_cache_store
 ```
 
-### SQL Caching
+You can access the cache by calling `Rails.cache`.
```

Comment:
This last sentence feels like it's asking for a code example :)

Reply:
Lol it does sound like that 😁. Done  ✅